deezer/utils.py

```diff
@@ -0,0 +1,12 @@
+"""Utils."""
+
+from collections import OrderedDict
+
+
+class SortedDict(OrderedDict):
```

Comment:
Can you make a note about why this is required? From what I understood, this is because `dict` ordering is deterministic before 3.6, and is not guaranteed in Python <3.7.

That means we can probably clean this up when we drop older versions.

Reply:
I add two comments and docstring in the new module.